packages/main/src/DatePicker.js

```diff
@@ -392,27 +392,23 @@ class DatePicker extends WebComponent {
 	}
 
 	/**
-	 * Formats a date according to the given format options.
-	 *
-	 * @param {Date} date The value to format
-	 * @param {boolean} utc Whether to use UTC
-	 * @returns {string} The formatted output value. If an invalid date is given, an empty string is returned
+	 * Gets a Date value corresponding to the current value property.
+	 * @returns {Date} The parsed value
 	 * @public
 	 */
-	format(date, utc) {
-		return this.getFormat().format(date, utc);
+	getDateValue() {
+		return this.getFormat().parse(this.value);
 	}
 
 	/**
-	 * Parses a string which is formatted according to the given format options.
-	 *
-	 * @param {string} value The string containing a formatted date value
-	 * @param {boolean} utc Whether to use UTC
-	 * @returns {Date} The parsed value
+	 * Sets the value property using a Date value,
+	 * formatted according to the given format options.
+	 * @param {Date} date The value to be set
 	 * @public
 	 */
-	parse(value, utc) {
-		return this.getFormat().parseValue(value, utc);
+	setDateValue(date) {
```

Comment:
it is confusing to set properties in two different ways, let's discuss this first.